gaming/xcom/config/BUILD

```diff
@@ -21,7 +21,7 @@
 
 load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 
-package(default_visibility = ["//github:__pkg__"])
+package(default_visibility = ["//gaming/xcom:__pkg__"])
```

Comment:
Git or github appears to be confused where the two almost identical files were moved. I think it matched them up in alphabetical order, `(github, xcom) -> (gaming/xcom, software/github)`.